pkg/vtpm/src/server.cpp

```diff
@@ -526,7 +526,7 @@ serveClient(int sock, sockaddr_in clientAddr)
             }
         }
         else {
-            cerr << "Failure processing the request, too large (maybe adjust MAX_PAYLOAD_SIZE)" << std::endl;
+            cerr << "Request size " << payloadSize << " is too large or small (maybe adjust MAX_PAYLOAD_SIZE)" << std::endl;
```

Comment:
payloadSize is uint32, does it make sense this error message ("... or small")? Which situations payloadSize can be too small?